src/snowcli/cli/nativeapp/manager.py

```diff
@@ -115,6 +118,34 @@ def find_row(cursor: DictCursor, predicate: Callable[[dict], bool]) -> Optional[
     )
 
 
+def _generic_sql_error_handler(
+    err: ProgrammingError, role: Optional[str] = None, warehouse: Optional[str] = None
+):
+    # Potential refactor: If moving away from python 3.8 and 3.9 to >= 3.10, use match ... case
+    if err.errno == 2043 or err.msg.__contains__(ERROR_MESSAGE_2043):
```

Comment:
A thought: what utility is the `__contains__` giving us?

Reply:
The CLI team has been using `__contains__` when checking for message in a `ProgrammingError` object. I followed that convention. e.g. `snowcli/src/snowcli/cli/snowpark/function/commands.py`